src/stackedchart/StackedChart.tsx

```diff
@@ -101,10 +127,13 @@ export default class StackedChart extends React.Component<
         const data = await getStackedChart({
             contact: this.props.contact,
             type: this.props.type,
+            fipsKeyForState: this.props.fipsKeyForState,
         });
-        if (data != null) {
+        if (data != null && data[0] != null && data[0].xAxisData.length > 0) {
```

Comment:
if js/ts, let's use `===` instead of `==`

Reply:
!= null, will check both `undefined` and `null`. !== null would check only for null.